### PR TITLE
Release Notes Heading Change

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_4.3.md [hunk 1]
```diff
@@ -381,7 +381,7 @@ init[5720]: [dh00000001 | dhcp-server-ns-1:1073742075] Command "/usr/sbin/ip net
 
 ## Release 4.3.3
 
-### New Features and Improvements
+### Resolved Issues Requiring Configuration Changes
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.3.md:384]: ugh, this was at a time when we were actually adding features to patches. This one should stay with New Features

File: docs/release_notes_128t_4.5.md [hunk 2]
```diff
@@ -155,7 +155,7 @@ OR
 
 **Release Date:** April 12, 2021 
 
-### New Features and Improvements
+### Resolved Issues Requiring Configuration Changes and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:158]: Did you mean to leave "and Improvements"?

File: docs/release_notes_128t_4.5.md [hunk 3]
```diff
@@ -173,7 +173,7 @@ OR
 
 ## Release 4.5.6
 
-### New Features and Improvements
+### Resolved Issues Requiring Configuration Changes and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:176]: Did you mean to leave "and Improvements"?

File: docs/release_notes_128t_4.5.md [hunk 4]
```diff
@@ -340,7 +340,7 @@ SSH Root Login is not permitted.
 Before upgrading, ensure that there is at least one user on each 128T system that has sudo privileges. Failure to do so may result in the loss of remote management connectivity to the 128T Networking Platform. Please see the [Installation Overview](intro_installation.md) for additional information. 
 :::
 
-### New Features and Improvements
+### Resolved Issues Requiring Configuration Changes and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:343]: Did you mean to leave "and Improvements"?

File: docs/release_notes_128t_4.5.md [hunk 5]
```diff
@@ -502,7 +502,7 @@ As part of the SSH hardening process, inactive SSH sessions will be logged out a
 
 ## Release 4.5.1
 
-### New Features and Improvements
+### Resolved Issues Requiring Configuration Changes and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_128t_4.5.md:505]: Did you mean to leave "and Improvements"?

File: docs/release_notes_128t_5.0.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ Alternatively, refer to the **[List of Releases](about_releases.mdx)** page for
 ## Release 5.0.1
 **Release Date:** April 12, 2021 
 
-### New Features and Improvements
+### Resolved Issues Requiring Configuration Changes and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.0.md:17]: Did you mean to leave "and Improvements"?

File: docs/release_notes_128t_5.2.md [hunk 7]
```diff
@@ -164,7 +164,7 @@ Contains fixes from: [Release 5.1.5, August 13, 2021](release_notes_128t_5.1.md#
 
 Contains fixes from: [Release 5.1.4, June 28, 2021](release_notes_128t_5.1.md#release-514)
 
-###  New Features and Improvements
+###  Resolved Issues Requiring Configuration Changes and Improvements
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.2.md:167]: Did you mean to leave "and Improvements"?

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by hadrielk — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**